lib/algos/list/delimiterCurrent.ts

```diff
@@ -32,12 +32,13 @@ class DelimiterCurrent extends Delimiter {
         this.excludedDataStoreName = parameters.excludedDataStoreName;
         // used for monitoring
         this.start = null;
-        this.evaluatedKeys = 0;
+        this.scannedKeys = 0;
     }
 
-    genMDParamsV1() {
-        const params = super.genMDParamsV1();
-
+    genMDParamsV0() {
+        const params = super.genMDParamsV0();
+        // lastModified and dataStoreName parameters are used by metadata that enables built-in filtering, 
```

Comment:
It feels odd to see those parameters now in the `genMDParamsV0` section, since MongoDB uses V1 metadata. But it's more theoretical as Metadata doesn't use them anyway (we could maybe remove them, but we'd have to add them back in the 8.x branches, so good for me, they should not have side-effects in Metadata).

Reply:
Could "query/built-in filter" be supported one day in Metadata?